magenta-lib/src/main/scala/magenta/deployment_type/CloudFormation.scala

```diff
@@ -34,6 +34,9 @@ object CloudFormation extends DeploymentType {
   val templateParameters = Param[Map[String, String]]("templateParameters",
     documentation = "Map of parameter names and values to be passed into template. `Stage` and `Stack` (if `defaultStacks` are specified) will be appropriately set automatically."
   ).default(Map.empty)
+  val templateStageParameters = Param[Map[String, Map[String, String]]]("templateStageParameters",
+    documentation = "Like templateParameters, but supports stage-specific configuration."
```

Comment:
I think we'd need a more detailed description, though I need to get round to revamping the documentation in RiffRaff to make this info more prominent.